Application/EdFi.Ods.AdminApi/Infrastructure/Database/Commands/AddApplicationCommand.cs

```diff
@@ -27,8 +27,8 @@ public AddApplicationCommand(IUsersContext usersContext, InstanceContext instanc
 
     public AddApplicationResult Execute(IAddApplicationModel applicationModel)
     {
-        var profile = applicationModel.ProfileId.HasValue
-            ? _usersContext.Profiles.SingleOrDefault(p => p.ProfileId == applicationModel.ProfileId.Value)
+        var profiles = applicationModel.ProfileIds != null
+            ? _usersContext.Profiles.Where(p => applicationModel.ProfileIds!.Contains(p.ProfileId))
```

Comment:
@jagudelo-gap 
I see Profile id list is nullable type. Adding "!" will convince the compiler. But there is a chance for getting a null value at runtime. So please make sure to check null.

Reply:
@CSR2017 
The check for null is in line 30

Reply:
Sorry Juan. Wow, how did I miss it !! THanks